### PR TITLE
[geolocator] Initial commit

### PR DESCRIPTION
**Pull request by bbrto21**
* Add basic structure of plugin
* Copy example from original

Signed-off-by: Boram Bae <boram21.bae@samsung.com>

### CODE REVIEW

**Review by bwikbs — APPROVED** [review]
:+1:

File: packages/geolocator/example/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# geolocator_android_example
+
+Demonstrates how to use the geolocator_android plugin.
```
Comment (WonyoungChoi) [inline packages/geolocator/example/README.md:3]:
```suggestion
Demonstrates how to use the geolocator_tizen plugin.
```

File: packages/geolocator/example/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    sdk: flutter
+  flutter_lints: ^1.0.4
+
+flutter:
+  uses-material-design: true
+  assets:
+    - res/images/baseflow_logo_def_light-02.png
+    - res/images/poweredByBaseflowLogoLight@3x.png
+    - packages/baseflow_plugin_template/logo.png
+    - packages/baseflow_plugin_template/poweredByBaseflow.png
```
Comment (WonyoungChoi) [inline packages/geolocator/example/pubspec.yaml:32]: please add newline to EOF

File: packages/geolocator/example/tizen/Runner.csproj.user [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup/>
+  <PropertyGroup>
+    <FlutterEmbeddingPath>/home/boram/Work/f-project/flutter-tizen/embedding/csharp/Tizen.Flutter.Embedding.csproj</FlutterEmbeddingPath>
+  </PropertyGroup>
+</Project>
```
Comment (WonyoungChoi) [inline packages/geolocator/example/tizen/Runner.csproj.user:7]: newline also

File: packages/geolocator/example/tizen/flutter/GeneratedPluginRegistrant.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
+//
```
Comment (bwikbs) [inline packages/geolocator/example/tizen/flutter/GeneratedPluginRegistrant.cs:1]: I guess we don't need these file

File: packages/geolocator/example/README.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# geolocator_android_example
```
Comment (WonyoungChoi) [inline packages/geolocator/example/README.md:1]:
```suggestion
# geolocator_tizen_example
```

**Comment by bbrto21** [comment]
I'm sad about the stupid CI :(

File: packages/geolocator/example/tizen/flutter/GeneratedPluginRegistrant.cs (hunk 4 above)
Comment (bbrto21) [reply packages/geolocator/example/tizen/flutter/GeneratedPluginRegistrant.cs:1]: Oh! thank you!

File: packages/geolocator/example/README.md (hunk 1 above)
Comment (bbrto21) [reply packages/geolocator/example/README.md]: Thanks! done!

File: packages/geolocator/example/pubspec.yaml (hunk 2 above)
Comment (bbrto21) [reply packages/geolocator/example/pubspec.yaml]: done

File: packages/geolocator/example/tizen/Runner.csproj.user (hunk 3 above)
Comment (bbrto21) [reply packages/geolocator/example/tizen/Runner.csproj.user]: done

File: packages/geolocator/example/README.md (hunk 5 above)
Comment (bbrto21) [reply packages/geolocator/example/README.md]: done

File: packages/geolocator/example/tizen/Runner.csproj.user (hunk 3 above)
Comment (bbrto21) [reply packages/geolocator/example/tizen/Runner.csproj.user]: flutter-tizen removes newlines in this file :cry:

Comment (WonyoungChoi) [reply packages/geolocator/example/tizen/Runner.csproj.user:7]: Yes, It seems that the `XMLDocument` of `package:xml` writes the file without EOF newline. :(

Comment (WonyoungChoi) [reply packages/geolocator/example/tizen/Runner.csproj.user:7]: I made a patch https://github.com/flutter-tizen/flutter-tizen/pull/185

Comment (swift-kim) [reply packages/geolocator/example/tizen/Runner.csproj.user:7]: Hey guys. Just don't check out the .user file.

@bbrto21 Maybe your project had been created before https://github.com/flutter-tizen/flutter-tizen/pull/178 was merged. Please run `flutter-tizen create` again and make sure the `.user` file is ignored by git.

**Comment by HakkyuKim** [comment]
I'm sure someone already asked, this is a template project correct? Implementations and documentations will be updated in a separate PR?

**Comment by bwikbs** [comment]
> I'm sure someone already asked, this is a template project correct? Implementations and documentations will be updated in a separate PR?

It's dev branch I guess :smile:

**Comment by HakkyuKim** [comment]
@bwikbs 
Oh, right!

**Review by HakkyuKim — APPROVED** [review]

**Pull request merged**